install/environment.root.yml

```diff
@@ -0,0 +1,32 @@
+###############################################################################
```

Comment:
This file should be mentioned as an alternative mode of installation in the section `https://github.com/zavolanlab/PAQR2/blob/9b1f6a38cbde0be33ced2d45b1c35b3b311864af/README.md#3-dependencies-installation` of the README.

Reply:
OK!